sdcm/utils/common.py

```diff
@@ -979,6 +979,29 @@ def tag_ami(ami_id, tags_dict, region_name):
     LOGGER.info("tagged %s with %s", ami_id, tags)
 
 
+def get_tables_with_compaction(tester, db_node, ks, request_timeout=300, non_compact=False):
+    tables = []
```

Comment:
Please add docstring to function added to utils.*, since they are going to be used by other parts of the code, they should be clear as what it aiming todo. And what the parameters means.